### ISSUE TITLE
can't find Fluorescent Fungus

### ISSUE THREAD

**Issue by oniiichannnn** [theBowja/genshin-db#74]
**Fluorescent Fungus** is a ascension material for some characters but I can't find it, what I've tried:
```js
let db = require("genshin-db")

db.materials("Fluorescent Fungus", { matchCategories: true })
```

**Comment by theBowja** [comment]
What is the version of the package you are using

**Comment by oniiichannnn** [comment]
> What is the version of the package you are using

3.10.3

**Comment by theBowja** [comment]
Fluorescent fungus is only available in v3.11.0+
The latest version is v3.13.1

Please update
